automatic/gimp/tools/chocolateyInstall.ps1

```diff
@@ -1,39 +1,15 @@
 ﻿$ErrorActionPreference = 'Stop';
 
+$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
 $packageArgs = @{
-  packageName    = 'gimp'
+  packageName    = $env:ChocolateyPackageName
   fileType       = 'exe'
+  file           = "$toolsPath\gimp-2.8.22-setup.exe"
   softwareName   = 'GIMP'
-
-  checksum       = 'a2e52129a28feec1ee3f22f5aaf9bdecbb02d51af6da408ace0a2ac2e0365c8b'
-  checksum64     = 'a2e52129a28feec1ee3f22f5aaf9bdecbb02d51af6da408ace0a2ac2e0365c8b'
-  checksumType   = 'sha256'
-  checksumType64 = 'sha256'
-
-  # The installer contains both 32-bit and 64-bit versions of GIMP, and will automatically use the appropriate one.
-  # Text quoted from <https://www.gimp.org/downloads/>, look below download buttons.
-  url            = 'https://download.gimp.org/mirror/pub/gimp/v2.8/windows/gimp-2.8.22-setup.exe'
-  url64bit       = 'https://download.gimp.org/mirror/pub/gimp/v2.8/windows/gimp-2.8.22-setup.exe'
-
-  silentArgs     = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-'
+  silentArgs     = "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /LOG=`"$($env:TEMP)\$($packageName).$($env:chocolateyPackageVersion).InnoSetup.log`""
```

Comment:
```
 /LOG=`"$( Join-Path (Get-PackageCacheLocation) InnoSetup.log )"`
```

Reply:
The idea is to log to a file in the same location as the ones used when running an msi installer, without creating uneeded directories. (I noticed I had included the chocolatey-core.extension though, which I need to remove)